samples/system-test/detect.v2beta1_test.js

```diff
@@ -0,0 +1,153 @@
+/**
+ * Copyright 2018, Google, LLC.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const test = require(`ava`);
+const {runAsync} = require('@google-cloud/nodejs-repo-tools');
+
+const cmd = 'node detect.v2beta1.js';
+//const testQuery = [`How do I sign up?`];
+const testQuery = `Where is my data stored?`;
+
+const testKnowledgeBaseName = 'TestKnowBase';
+
+let knowbaseDisplayName;
+let knowbaseFullName;
+
+let documentFullPath;
+const testDocName = `TestDoc`;
+//const testDocumentPath = `https://dialogflow.com/docs/concepts/google-projects-faq`;
+const testDocumentPath = `https://cloud.google.com/storage/docs/faq`;
+
+test.before(`It should create a knowledge base`, async t => {
+  const output = await runAsync(
+    `${cmd} createKnowledgeBase -k ${testKnowledgeBaseName}`
+  );
+  const parsedOut = output.split(`\n`);
```

Comment:
❗️ **#BLOCKER**

Remove all of this "parsing" (split on newlines) code from the tests

Please `t.true(output.includes())` like most existing Node.js sample system tests

Reply:
For example, the `'It should detect Intent with Model Selection'` test below this LGTM – it looks like all of our other tests. Very simple. It calls the command and verified relevant content appeared in the console.log'd output.